src/libraries/System.Security.Cryptography.X509Certificates/src/Internal/Cryptography/ICertificatePal.cs

```diff
@@ -10,6 +10,28 @@
 
 namespace Internal.Cryptography
 {
+    internal struct PolicyData
+    {
+        internal byte[] ApplicationCertPolicies;
+        internal byte[] CertPolicies;
+        internal byte[] CertPolicyMappings;
+        internal byte[] CertPolicyConstraints;
+        internal byte[] EnhancedKeyUsage;
+        internal byte[] InhibitAnyPolicyExtension;
+
+        internal PolicyData(byte[] applicationCertPolicies = null, byte[] certPolicies = null,
```

Comment:
I don't see any usage of this ctor, why is it here? (I was going to ask "why is everything defaulted", but "why is it here at all" is up first)

Reply:
without it I was getting compilation errors about uninitialized variables/fields.

Reply:
But nothing's calling it... the places that create this type all start by assigning it to default.

Even if it's defined, since nothing said `new` nothing's calling this.

Reply:
I only add this because I was getting compilation error. I took it out, did full clean build on two platforms and I cannot reproduce it. It is truly not needed and so I removed it.